apps/web/src/services/iam/store/user-group-page-store.ts

```diff
@@ -9,25 +9,36 @@ import type { UserGroupListItemType } from '@/schema/identity/user-group/type';
 
 import type { ModalState } from '@/services/iam/types/user-group-type';
 
+interface iState {
```

Comment:
I think it would be good to align it by specifying the type using the store name!